content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +84,145 @@ names in the `train` array below:
 }
 ```
 
-## Plot templates (data series only)
+## Configuring a plot
+
+In order to create visualizations, users need to provide the data and
+(optionally) configuration that will help customize the plot. DVC provides two
+ways to configure visualizations. Users can mark specific <abbr>stage</abbr>
+<abbr>outputs</abbr> as plot or define plot configuration inside `dvc.yaml`
+under `plots` key.
+
+### Standalone plots
+
+Plots can be defined in `dvc.yaml` under the `plots` key. Unlike
+[stage plots](#stage-plots), these are especially useful when users want to
+compare data from different data sources residing on the same version of the
+project. For example, comparing training versus test results on current branch.
+
+### Stage plots
+
+When using `dvc run` or `dvc stage add`, instead of using
+`--outs/--outs-no-cache` particular outputs can be marked with
+`--plots/--plots-no-cache`. This will tell DVC that they are intended for
+visualizations.
+
+Upon running `dvc plots show/diff` DVC will collect stage plots alongside the
+standalone plots and display them conforming to their configuration. Note, that
+if there are stage plots in the project and they are also used in some plots
+standalone plots, DVC will create separate rendering for the stage plots and all
+standalone plots using them.
```

Comment:
I find this phrase a little confusing. Do we want to explain something like that the actual data files can be reused in the standalone plots and the stage plots?

Reply:
Yeah the terminology is confusing. Issues I see:

**Configuring a plot**

Why is is separate from `plots modify` and **Plot templates**? Those are 2 actual ways to configure plots (already slightly confusing that there's 2 plot configs, let's avoid a 3rd). I think we're talking about "defining plots" here...

**Stand-alone plots**

They're linked to a stage via an `output` name though, right? So why stand-alone? I think this is about reusable, pipeline-level definitions.

**Stage plots**

Sounds like we're adding another dimension to the concept of "plot types" (dimension 1 being data-series vs. images) which may be confusing. And again, even pipeline-level plot definitions are linked to stages via an `output` name, so the term isn't accurate.

> Will try to post suggestions once I get a better understanding of the big picture...

Reply:
OK here's what I'm thinking: This ref is now way too long. And even when clearly [we need to make a proper guide](https://github.com/iterative/dvc.org/issues/2572), a lot of the detail here can be compacated and reorganized. Specific plan of action I recommend:

**Configuring a plot** / **Stand-alone plots**/ **Stage plots**

Make this a paragraph or 2 in the existing **Description** of this page (ideally no H3 sections, or make a single **Defining plots** section if absolutely necessary). It probably replaces **Supported file formats** since that info is included in the plot definition and syntax blocks.

Terminology-wise, I'd go with descriptive phrasing like "plots defined in stages", "found in the `plots` field of `dvc.yaml`", etc. instead of coming up with special terms. From the user-perspective all plots are the same once rendered.

**Syntax (standalone plots only)**

Move all of this into [**Standalone plots configurations**](https://dvc-org-7086-flexible-p-d0hfxv.herokuapp.com/doc/user-guide/project-structure/pipelines-files#standalone-plots-configurations) (in the dvc.yaml schema guide) and link from this page as needed.

> I'd rename that section to **Reusable plot definitions** (will leave a specific suggestion ⏳)

Try to summarize as much as possible. Use tables if needed. I can jump in to help with that once the content is in the right place. Thanks